depends/README.md

```diff
@@ -122,7 +122,8 @@ Common `host-platform-triplet`s for cross compilation are:
 
 - `i686-pc-linux-gnu` for Linux x86 32 bit
 - `x86_64-pc-linux-gnu` for Linux x86 64 bit
-- `x86_64-w64-mingw32` for Win64
+- `x86_64-w64-mingw32` for Windows using MSVCRT
```

Comment:
How will a user know what to pick (here and below), given we don't explain the difference? Should one be marked as deprecated/unsupported? Probably easier to wait until we only support one, and then we don't need to worry about this.

Reply:
Given that full UCRT-capable toolchains are not yet available across all major distributions, developers on Ubuntu, for example, may still want to build for `x86_64-w64-mingw32`.

Reply:
> Ah ok, so maybe someone could open a question/request at https://launchpad.net/ubuntu/+source/gcc-mingw-w64/+bugs, so that the progress can be tracked.

https://bugs.launchpad.net/ubuntu/+source/gcc-mingw-w64/+bug/2132114

Reply:
> > developers on Ubuntu, for example, may still want to build for x86_64-w64-mingw32.
> 
> Are we supporting that, if the CI is removed in #33593 (along with dropping workarounds going forward).

We won't support that once workarounds are dropped and code become incompatible with MSVCRT. By that point, I don’t anticipate any issues with providing that support.

Reply:
> By that point, 

Isn't that point #33593? The workarounds should be removed along with the CI being dropped, otherwise they are untested and unused.

Reply:
> Isn't that point #33593? The workarounds should be removed along with the CI being dropped, otherwise they are untested and unused.

It could be in #33593 or in a follow-up PR.